provisioning/service/src/Enrollments/AttestationMechanism.cs

```diff
@@ -48,42 +48,6 @@ internal AttestationMechanism(Attestation attestation)
             }
         }
 
-        [JsonConstructor]
```

Comment:
How do we know this one is safe to remove? It is a constructor for JSON so it wouldn't have references.

Reply:
Yeah, the failed e2e tests complained about the same. I will undo the `JsonConstructor` removing.